src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerPushImage.groovy

```diff
@@ -36,6 +36,13 @@ class DockerPushImage extends AbstractDockerRemoteApiTask implements RegistryCre
     @Optional
     String tag
```

Comment:
Should we deprecate this?

Reply:
I think so. I've been throwing around an idea of bumping the plugin to `4.0.0` and with that include a refactoring of a handful of tasks to allow for multiple things to be worked on (i.e. multiple execs, multiple pushes, etc..).